user-guide/consumers.md

```diff
@@ -200,14 +200,35 @@ Consumer.start('account:command')
 
 In the above example, the consumer's position stream would be `account:command-someConsumer`.
 
+Consumers can also be assigned an identifier when they are started. If an identifier macro is also declared on the consumer class, the one given when starting the consumer is chosen.
+
+``` ruby
+Consumer.start('account:command', identifier: 'otherIdentifier')
+```
+
+## Conditions
+
+Since the consumer reads the given stream using a SQL query, that query can be extended by the `condition` keyword argument. This further constrains the messages read by the consumer beyond selecting only the messages of the stream being consumed. For instance, this allows a consumer to only read messages of a given type, or only a subset of the streams within a category for parallel processing across multiple consumers.
+
+```ruby
+SomeConsumer.start('someCategory', condition: "hash64(stream_name) % 2 = 0")
+SomeConsumer.start('someCategory', condition: "hash64(stream_name) % 2 = 1")
+```
+
+In the above example, two instances of the same consumer have been started, each consuming different entity streams within the category stream `someCategory`.
+
+::: danger
+Usage of this feature should be treated with caution, as it can cause messages to be read by consumers in a different order in which they were written. Idempotence techniques which rely on processing messages in-order therefore may not work in conjuction with `conditions`. Therefore, it is recommended that this feature be used with care, and only in cases where the ramifications are fully understood.
+:::
+
```

Comment:
The example isn't super clear, but I couldn't think of a simpler example that was "safe" to use in applicative code. I wonder if we should omit an example.

Reply:
Actually, I'm wondering if correlation stream category would make more sense here. It's a common use case for pub/sub, and the most likely reason to use this feature.

Reply:
👍